pandas/tseries/tests/test_daterange.py

```diff
@@ -520,15 +520,28 @@ def test_range_closed_with_tz_aware_start_end(self):
             self.assert_index_equal(expected_left, left)
             self.assert_index_equal(expected_right, right)
 
-        # test with default frequency, UTC
-        begin = Timestamp('2011/1/1', tz='UTC')
-        end = Timestamp('2014/1/1', tz='UTC')
+        begin = Timestamp('2011/1/1')
+        end = Timestamp('2014/1/1')
+        begintz = Timestamp('2011/1/1', tz='US/Eastern')
+        endtz = Timestamp('2014/1/1', tz='US/Eastern')
+
+        for freq in ["1D", "3D", "2M", "7W", "3H", "A"]:
+            closed = date_range(begin, end, closed=None, freq=freq,
+                                tz='US/Eastern')
+            left = date_range(begin, end, closed="left", freq=freq,
+                              tz='US/Eastern')
+            right = date_range(begin, end, closed="right", freq=freq,
+                               tz='US/Eastern')
+            expected_left = left
+            expected_right = right
 
-        intervals = ['left', 'right', None]
-        for i in intervals:
-            result = date_range(start=begin, end=end, closed=i)
-            self.assertEqual(result[0], begin)
-            self.assertEqual(result[-1], end)
```

Comment:
I removed this test, as I think it is testing the wrong behaviour (it tests that the different closed options give all the same result, which should not be (and is not the case with naive timestamps))